docs/demo/pydata.md

```diff
@@ -0,0 +1,88 @@
+---
+file_format: mystnb
+kernelspec:
+  name: python3
+---
+
+# PyData Library Styles
+
+This theme has built-in support and special styling for several major visualization libraries in the PyData ecosystem.
+This ensures that the images and output generated by these libraries looks good for both light and dark modes.
+Below are examples of each that we use as a benchmark for reference.
+
+## Pandas
+
+```{code-cell}
+import numpy as np
+import pandas as pd
+
+data = np.random.randn(100, 3)
```

Comment:
This is a nitpick not related to the CSS but important IMO. We should not advertise for old syntax. This should be replaced with
```suggestion
rng = np.random.default_rng()
data = rng.standard_normal((100, 3))
```